CONTRIBUTING.md

```diff
@@ -15,10 +15,16 @@ photo:
 categories: [Other]
 ```
 
-The slug should match the file name. Generally, the `date` should be similar to the file name, but won't be identical - name the file according to when you created the blog post, but make sure you update the date when you publish the post.
+The slug should match the file name. Generally, the `date` should be similar to the file name, but won't be identical - name the file according to when you created the blog post using the format `yyyy-mm-post-slug`, but make sure you update the date in the YAML header when you publish the post.
 
 Categories should be one (or more of): "case studies", "learn", "package", "programming", or "other".
 
+### Filetype
+
+Posts can be written in any of the blogdown-supported formats: `.Rmd`, `.Rmarkdown`, and `.md` (in the event that you do not have any code in your post). 
```

Comment:
I think we should categorical recommend against `.Rmd` in the interest of keeping things simple